regression_test.go

```diff
@@ -149,3 +137,34 @@ func issue1(g *G, itest int) {
 	}
 	g.Assert(value).Equal(expectedValue)
 }
+
+func issue1(g *G) {
+	// Check with slice value for attribute.
+	attrs := Attributes{
+		"id":                  "user-employee-123456789",
+		"loggedIn":            true,
+		"employee":            true,
+		"country":             "france",
+		"dietaryRestrictions": []string{"gluten_free"},
+	}
+
+	issue1Like(g, attrs)
+}
+
+func issue5(g *G) {
+	// Check with array value for attribute.
+	attrs := Attributes{
+		"id":                  "user-employee-123456789",
+		"loggedIn":            true,
+		"employee":            true,
+		"country":             "france",
+		"dietaryRestrictions": [1]string{"gluten_free"},
+	}
+
+	issue1Like(g, attrs)
+
+	// Check that there's no problem using a nil context.
+	var nilContext *Context
+	gbTest := New(nilContext)
+	g.Assert(gbTest.Enabled()).IsTrue()
```

Comment:
Can we also check the expected value? It should probably be whatever the `defaultValue` property is on the JSON, if I'm not mistaken.

Reply:
I think I made this confusing by refactoring it... The expected value is tested at https://github.com/growthbook/growthbook-golang/blob/5-avoid-nil-context/regression_test.go#L138 which is in the `issue1Like` function, which I factored out because issue #1 and issue #5 have more or less identical tests.

I can probably make it less confusing by completely separating out the nil context test. Would that help, do you think?

Reply:
Oh, yeah, I think that would be great. We don't need to be DRY in the tests. Probably best not to be (the less logic in tests the better). It would be great to read all assertions in the test block without having to go to another function.

Reply:
Just pushed some changes that should make things clearer — I de-DRYed those two tests, and split the nil context test out as well. Is that better?